### ISSUE TITLE
Verify access to admin from Reticulum server.

### ISSUE THREAD

**Issue by yalegria** [RutgersGRID/hubs#32]

**Comment by Voxelghiest** [comment]
In testing the functionality of the Hubs Admin panel, I ran into an issue. Even though the admin panel's landing page loaded correctly, anytime I tried to access the tabs on the side (Accounts, Scenes, etc.), it kicked me out and returned me to the home page for Hubs Client.

The GET requests that Hubs Admin made to the PostgREST server that handles accessing the data for Hubs were returning error code 502. However, after finding [this post](https://github.com/mozilla/hubs/issues/4970#issue-1087523703), where someone had a similar issue, I was able to fix the problem.

When I configured the PostgREST server for Hubs Admin, I used [this tutorial](https://github.com/mozilla/hubs-ops/wiki/Running-PostgREST-locally) that Mozilla has on their Hubs-Ops repo. The tutorial required me to generate a file, `reticulum-jwk.json`, and provided two options to do so: either by using the iex console while Reticulum is running, or by downloading the file from Mozilla's servers.

I chose to download their version of the file, but when I went back and redid the process by creating the file myself in the iex console, it fixed the issue.
